tensorflow_datasets/core/features/array_2d.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+# Copyright 2021 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Array2D feature for raising error (Not implemented currently). """
```

Comment:
Array2d can already be represented by features.Tensor or features.Image. We don't want a new class in the future. If we wanted to update the huggingface wrapper, we would map Array2d -> Image